src/sources/soundsourceproxy.h

```diff
@@ -22,29 +24,41 @@ class SoundSourceProxy {
     /// registered.
     static bool registerProviders();
 
-    static QStringList getSupportedFileExtensions() {
-        return s_soundSourceProviders.getRegisteredFileExtensions();
+    static QStringList getSupportedFileTypes() {
+        return s_soundSourceProviders.getRegisteredFileTypes();
     }
     static const QStringList& getSupportedFileNamePatterns() {
         return s_supportedFileNamePatterns;
     }
     static const QRegularExpression& getSupportedFileNamesRegex() {
         return s_supportedFileNamesRegex;
     }
+    static QString getFileTypeByMimeType(const QMimeType& mimeType) {
```

Comment:
The naming of the function and member variable is slightly inconsistent with `getFileSuffixesForFileType()`. It shouldn't matter that one returns a single item while the other returns multiple items.